libs/ui/src/examples/Fetch/E3_FetchRequest.tsx

```diff
@@ -1,16 +1,29 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { useEffect, useState } from 'react'
 
-import { Box, Stack } from '@chakra-ui/react'
+import { Box, Stack, Text } from '@chakra-ui/react'
 import { API_URL, TOKEN } from '@wsvvrijheid/config'
 import { request } from '@wsvvrijheid/utils'
 
 export const FetchRequest = () => {
   const [blogs, setBlogs] = useState([])
-
+  const getBlogs = async () => {
+    const { data } = await request({ url: 'api/blogs' })
```

Comment:
I always recommend specifying type of the data.

```suggestion
    const { data } = await request<Blog[]>({ url: 'api/blogs' })
```